package.json

```diff
@@ -57,6 +64,7 @@
     "format-fix": "prettier --write \"src/**/*.js\"",
     "start": "node --require esm make dev -s -w -d data",
     "build": "npm run lib",
+    "build-webpack": "webpack --config config/webpack.config.dev.js",
```

Comment:
I have to run 'npm run build' (which fails) before I run this target for things to work, I think that this should be patched up a little better than this if we can.